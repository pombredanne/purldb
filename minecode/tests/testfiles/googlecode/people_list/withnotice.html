


<!DOCTYPE html>
<html>
<head>
 <link rel="icon" type="image/vnd.microsoft.icon" href="http://www.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"assetHostPath":"http://www.gstatic.com/codesite/ph","projectName":"zffl","projectHomeUrl":"/p/zffl","urlPrefix":"p","domainName":null,"relativeBaseUrl":"","profileUrl":null,"token":null,"assetVersionPath":"http://www.gstatic.com/codesite/ph/11113157398175881470","absoluteBaseUrl":"http://code.google.com","loggedInUserEmail":null};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 
 </script>
 
 
 <title>People - 
 zffl -
 
 
 Learn - Google Project Hosting
 </title>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOARCHIVE">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11113157398175881470/css/ph_core.css">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11113157398175881470/css/ph_list.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11113157398175881470/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 #resultstable td, #resultstable th { padding-right:2em }

 
 </style>
</head>
<body class="t1">
<script type="text/javascript">
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=http%3A%2F%2Fcode.google.com%2Fp%2Fzffl%2Fpeople%2Flist&amp;followup=http%3A%2F%2Fcode.google.com%2Fp%2Fzffl%2Fpeople%2Flist" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->

 <div style="font-weight:bold; color:#a03; padding:5px; margin-top:10px; text-align:center; background:#ffeac0;">
 Project Hosting will be READ-ONLY <a href='http://www.timeanddate.com/worldclock/fixedtime.html?month=8&day=9&year=2011&hour=9&min=00&sec=0&p1=64'>Tuesday at 9:00am CDT</a> for brief maintenance.
 
 </div>




 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0">
 <tr style="height: 58px;">
 
 <td id="plogo">
 <a href="/p/zffl/">
 
 <img src="http://www.gstatic.com/codesite/ph/images/defaultlogo.png" alt="Logo">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 <div id="pname">
 <a href="/p/zffl/">zffl</a>
 </div>
 
 <div id="psum">
 <a id="project_summary_link" href="/p/zffl/" >Learn</a>
 
 </div>
 
 
 </td>
 <td style="white-space:nowrap;text-align:right; vertical-align:bottom;">
 
 <form action="/hosting/search">
 <input size="30" name="q" value="" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 
<div id="mt" class="gtb"> 
 <a href="/p/zffl/" class="tab active">Project&nbsp;Home</a>
 
 
 
 
 
 
 
 
 
 
 
 
 <div class=gtbc></div>
</div>
<table cellspacing="0" cellpadding="0" width="100%" align="center" border="0" class="st">
 <tr>
 
 
 
 
 
 
 
 <td class="subt">
 <div class="st3">
 <div class="isf">
 <span class="inst1">
 <a href="/p/zffl/">Summary</a>
 </span>
 
 
 &nbsp;
 <span class="inst2">
 <a href="/p/zffl/updates/list">Updates</a>
 </span>
 
 
 &nbsp;
 <span class="inst3">
 <a href="/p/zffl/people/list">People</a>
 </span>
 
 
 </div>
</div>

 </td>
 
 
 <td align="right" valign="top" class="bevel-right"></td>
 </tr>
</table>


<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 
<!-- IE -->





<script type="text/javascript">
function _showBelow(){}
function _toggleStar(){}
function _rowRolloverOn(){}
function _rowRolloverOff(){}
function _goIssue(){}
function _goFile(){}
</script>

<script type="text/javascript">
 var cancelBubble = false;
</script>
<form method="POST" action="addMembers.do"
 >
<div id="colcontrol">
 <div class="list">
 <div class="pagination">
 
 1 - 1 of 1
 
 </div>
 <b>Project People</b>
 &nbsp; &nbsp;
 
 </div>
 <table cellspacing="0" cellpadding="2" border="0" class="results" id="resultstable" width="100%">
 <tbody>
 
 <tr id="headingrow">
 
 
 <th style="white-space:nowrap">Name</th>
 
 <th style="white-space:nowrap">Role</th>
 
 <th style="white-space:nowrap">Duties</th>
 
 <th style="white-space:nowrap">Notes</th>
 
 </tr>
 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/zffl/people/detail?u=flxxxxfl')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/zffl/people/detail?u=flxxxxfl"
 >flxxxxfl</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/zffl/people/detail?u=flxxxxfl')"
 ><a onclick="cancelBubble=true" href="/p/zffl/people/detail?u=flxxxxfl"
 >
 
 
 Owner
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/zffl/people/detail?u=flxxxxfl')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/zffl/people/detail?u=flxxxxfl">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/zffl/people/detail?u=flxxxxfl')"
 ><a onclick="cancelBubble=true" href="/p/zffl/people/detail?u=flxxxxfl"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 
 </tbody>
 </table>
 <div class="list-foot">
 <div class="pagination">
 
 1 - 1 of 1
 
 </div>
 &nbsp;
 </div>
</div>




</form>

<script type="text/javascript" src="http://www.gstatic.com/codesite/ph/11113157398175881470/js/dit_scripts.js"></script>

 
 
 
 <script type="text/javascript" src="http://www.gstatic.com/codesite/ph/11113157398175881470/js/ph_core.js"></script>
 
 
 
 
 <script type="text/javascript" src="/js/codesite_product_dictionary_ph.pack.04102009.js"></script>
</div> 
<div id="footer" dir="ltr">
 <div class="text">
 &copy;2011 Google -
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>
 
 


 
 </body>
</html>

