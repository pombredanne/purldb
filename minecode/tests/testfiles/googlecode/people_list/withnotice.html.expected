([{'detail_url': u'http://code.google.com/u/flxxxxfl',
   'person_id': u'flxxxxfl',
   'role': u'owner',
   'username': u'flxxxxfl'}],
 [])